src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -252,7 +252,25 @@ def __init__(self, target: str, options: ChannelArgumentType,
         calls = []
         call_tasks = []
         for task in tasks:
-            stack = task.get_stack(limit=1)
+            try:
+                stack = task.get_stack(limit=1)
+            except AttributeError as attribute_error:
+                # NOTE(lidiz) tl;dr: If the Task is created with a CPython
+                # object, it will trigger AttributeError.
+                #
+                # In the global finalizer, the event loop schedules
+                # a CPython PyAsyncGenAThrow object.
+                # https://github.com/python/cpython/blob/00e45877e33d32bb61aa13a2033e3bba370bda4d/Lib/asyncio/base_events.py#L484
+                #
+                # However, the PyAsyncGenAThrow object is written in C and
+                # failed to include the normal Python frame objects. Hence,
+                # this exception is a false negative, and it is safe to ignore
+                # the failure (someone should fix it in CPython!).
+                # https://github.com/python/cpython/blob/a025d4ca99fb4c652465368e0b4eb03cf4b316b9/Objects/genobject.c#L1989
+                if 'frame' in str(attribute_error):
```

Comment:
Can we add a TODO with a Github issue pointing out that we should remote the workaround once we drop support for 3.7? That will be years from now, so it's good to document that debt.

Reply:
It will be available for `3.8.3` and `3.9`, so we will have to wait for the drop support of `3.8`. It's definitely years to come... Since an GitHub issue isn't going to last that long, I think comment in code is better. Let's hope someone will update this path 5-10 years later.

Reply:
Fair enough. The comment will be sufficient. But can we at least add a `TODO` without an issue? That way, `grep -rn TODO src` will turn this up.

Reply:
A line of TODO added. PTALAA.